src/obb_demo/boardground.cljs

```diff
@@ -47,13 +47,22 @@
   (let [dir (element/element-direction element)]
     (first (name dir))))
 
+(defn- process-unit-name
+  "If the unit name needs transformations, does it to match the resources
+  on the main site."
+  [unit-name]
+  (if (= unit-name "star")
+    "flag"
+    unit-name))
```

Comment:
This is a general comment, but I think this code is temporary and we will want to remove it at some point. I think it would be a good idea to document it in some way. I had to do this a couple of times in the past and the way I decided to do so was to open an issue, put a comment on the code and the commit message referring to that issue.

I don't know what you think but this kinds of things are not easy to remember and doing so can save a bit of time to the person that has to handle this.

My rule of thumb is to do this when a change is not transparent at all. By transparency I mean that when a change is required, I can easily determine what are the things that I need to do and the code that I need to update in order to achieve my goal.

Let me know what you think.